gdk4-macos/Cargo.toml

```diff
@@ -22,6 +22,7 @@ gdk.workspace = true
 gio.workspace = true
 glib.workspace = true
 libc.workspace = true
+cocoa = "0.26"
```

Comment:
Same like we do for all external crates integrations, it has to be optional.

Reply:
done